Adjust Paddings, other fixes

darthwing-duck opened the pull request:
## What
This just adjusts some of the paddings for smaller screens, specifically the mission statement and join statement areas could look a little empty or a little cramped depending on how small the screen is. 

- Join already had some work and everything looks good. 
- This reduces the mission to just the short form.
- Update the alt text of the benefit image.
- move / delete some images to improve build speed/site size.

Specifically:

Before
![image](https://user-images.githubusercontent.com/3760863/281573496-f86d10f9-affa-4f94-a1e5-2759109d87a3.png)

After:
![image](https://github.com/mkpjapan/mkpjapan.github.io/assets/3760863/69a619de-5b3a-4c54-97d0-57dce1e4528f)

Join has some issues too:

Before:
![image](https://user-images.githubusercontent.com/367876/281615419-fd3a382c-695c-4e00-b1bd-b195e2651bb9.png)

After:
![image](https://github.com/mkpjapan/mkpjapan.github.io/assets/3760863/0aa498a7-cfa9-4b92-b6f8-54063fe7ec9b)

lunardog commented:
This could be better:
<img width="401" alt="Screenshot 2023-11-09 at 13 25 22" src="https://github.com/mkpjapan/mkpjapan.github.io/assets/367876/fd3a382c-695c-4e00-b1bd-b195e2651bb9">

I used "Responsive mode" in the Firefox Inspector and adjusted the width to <400px.
<img width="227" alt="Screenshot 2023-11-09 at 13 24 53" src="https://github.com/mkpjapan/mkpjapan.github.io/assets/367876/0390322b-157e-4946-9392-7eba663312d6">

darthwing-duck commented:
> This could be better: <img alt="Screenshot 2023-11-09 at 13 25 22" width="401" src="https://user-images.githubusercontent.com/367876/281615419-fd3a382c-695c-4e00-b1bd-b195e2651bb9.png">
> 
> I used "Responsive mode" in the Firefox Inspector and adjusted the width to <400px. <img alt="Screenshot 2023-11-09 at 13 24 53" width="227" src="https://user-images.githubusercontent.com/367876/281615438-0390322b-157e-4946-9392-7eba663312d6.png">

Fixed in latest.
